utils/check_deps.js

```diff
@@ -128,14 +128,24 @@ async function innerCheckDeps(root) {
         importPath = path.resolve(path.dirname(fileName), importName);
       } else if (importName.startsWith('@')) {
         const tokens = importName.substring(1).split('/');
-        const package = tokens[0];
-        if (packages.has(package))
-          importPath = packages.get(package) + tokens.slice(1).join('/');
+        if (packages.has(`@${tokens[0]}/${tokens[1]}`)) {
+          // Namespaced package.
+          importPath = packages.get(`@${tokens[0]}/${tokens[1]}`) + tokens.slice(2).join('/');
```

Comment:
Why is this different from other packages, e.g. `playwright-core`? I'd leave `importPath` undefined and go the line 162 old code instead.